cmd/porter/bundle.go

```diff
@@ -163,9 +163,9 @@ For example, the 'debug' driver may be specified, which simply logs the info giv
 		"Path to the porter manifest file. Defaults to the bundle in the current directory.")
 	f.StringVar(&opts.CNABFile, "cnab-file", "",
 		"Path to the CNAB bundle.json file.")
-	f.StringSliceVarP(&opts.ParameterSets, "parameter-set", "p", nil,
+		f.StringArrayVarP(&opts.ParameterSets, "parameter-set", "p", nil,
 		"Name of a parameter set for the bundle. It should be a named set of parameters and may be specified multiple times.")
-	f.StringSliceVar(&opts.Params, "param", nil,
+	f.StringArrayVar(&opts.Params, "param", nil,
```

Comment:
:+1:  great, and you already saw we have different commands that use this, so we need this for them also, for example invoke https://github.com/getporter/porter/pull/1931/files#diff-9fc541206435fd5b66cd057759dc4fd2ccf36aed05594a186148f90cb57caf75R282